theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -1699,3 +1699,157 @@ def test_conv2d_grad_wrt_weights(self):
                                                                                                   )
                         f_new = theano.function([self.x, self.output_grad_wrt], conv_wrt_w_out)
                         utt.assert_allclose(f_new(input_val, out_grad_val), f_old(input_val, filter_val, out_grad_val))
+
+
+class Grouped_conv_noOptim(unittest.TestCase):
+    conv2d = staticmethod(theano.tensor.nnet.abstract_conv.AbstractConv2d)
+    conv2d_gradw = staticmethod(theano.tensor.nnet.abstract_conv.AbstractConv2d_gradWeights)
+    conv2d_gradi = staticmethod(theano.tensor.nnet.abstract_conv.AbstractConv2d_gradInputs)
+    conv2d_op = staticmethod(theano.tensor.nnet.abstract_conv.AbstractConv2d)
+    conv2d_gradw_op = staticmethod(theano.tensor.nnet.abstract_conv.AbstractConv2d_gradWeights)
+    conv2d_gradi_op = staticmethod(theano.tensor.nnet.abstract_conv.AbstractConv2d_gradInputs)
```

Comment:
Why are the _op versions of attribute the exact same thing as the non _op variants?

Why do you need to wrap ops in `staticmethod`?